lib/Mojo/IOLoop.pm

```diff
@@ -646,6 +657,12 @@ seconds.
     ...
   });
 
+=head2 transition
+
+  my $stream = Mojo::IOLoop->transition($id, 'Mojo::IOLoop::Stream::HTTPClient');
+
+Replace connection with new one from different class.
```

Comment:
Maybe something simple like `Transition stream to a different class.` would be better?